app/features/executive/availability.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Availability
+
+  As an executive
+  I want the CIIP portal to be available at all times
+  So that the service design is designed effectively
```

Comment:
I'm not sure I quite understand the explicitness behind this reasoning?

Reply:
the intent here is that it's difficult to deliver an effective service if the service is offline; updated